### PR TITLE
MAISTRA-562 fix README to tell the Kiali Operator installer not to also install Kiali

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

**Comment by jmazzitelli** [comment]
cc @rcernich

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ oc create -n observability -f https://raw.githubusercontent.com/jaegertracing/ja
 To install the Kiali operator, execute the following command:
 
 ```
-bash <(curl -L https://git.io/getLatestKialiOperator) --operator-image-version v1.0.0 --operator-watch-namespace '**' --accessible-namespaces '**'
+bash <(curl -L https://git.io/getLatestKialiOperator) --operator-image-version v1.0.0 --operator-watch-namespace '**' --accessible-namespaces '**' --operator-install-kiali false
```
Comment (jwendell) [inline README.md:28]: Are the double stars correct?

**Review by jwendell — CHANGES_REQUESTED** [review]
LGTM. Could you please create a JIRA and change the PR title to include the Jira id?

**Comment by jmazzitelli** [comment]
> Could you please create a JIRA and change the PR title to include the Jira id

done

File: README.md (hunk 1 above)
Comment (jmazzitelli) [reply README.md:28]: > Are the double stars correct?

yes

**Review by jwendell — APPROVED** [review]

**Pull request merged**